calico-enterprise/reference/resources/felixconfig.mdx

```diff
@@ -305,10 +306,19 @@ failsafeOutboundHostPorts:
 | DelayDNSResponse  | Felix delays any DNS response until related IPSets are programmed. This introduces some latency to all DNS packets (even when no IPSet programming is required), but it ensures policy hit statistics are accurate. This is the recommended setting when you are making use of staged policies or policy rule hit statistics. |
 | NoDelay           | Felix does not introduce any delay to the packets. DNS rules may not have been programmed by the time the first packet traverses the policy rules. Client applications need to handle reconnection attempts if initial connection attempts fail. This may be problematic for some applications or for very low DNS TTLs.      |
 
-On Windows, or when using the eBPF dataplane, this setting is ignored and `NoDelay` is always used.
+On Windows, or when using the eBPF dataplane, this setting is ignored. Windows always use `NoDelay` while eBPF has its own `BPFDNSPolicyMode` option.
 
 A linux kernel version of 3.13 or greater is required to use `DelayDNSResponse`. For earlier kernel versions, this value is modified to `DelayDeniedPacket`.
 
+### BPFDNSPolicyMode
+
+| Value             | Description                                                                                                                                                                                                                                                                                                                   |
+| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Inline            | Felix does not introduce any delay to the packets. DNS responses are parsed inline with other bpf processing and programs policy rules instantly before the DNS response is released for further processing. Only suffix domain wildcards are supported, that is `*.x.y.z` is ok, but `x.*.y.z` is not acceptable with this option.|
```

Comment:
I've tweaked a bit for grammar and readability.  Main changes are: clarify behavior for midfix wildcards (please confirm I'm right that it will fall back, rather than being unsupported) move mixed voices to mostly active voice, s/suffix/prefix/ (since we're talking about the wildcard, not the non wildcarded part), and some minor wording changes.

"Felix does not introduce any delay to any packets.  Felix's eBPF programs parse DNS responses and program policy rules immediately, before the DNS response is passed to the application.  This only applies to wildcard prefixes: *.x.y.z will be processed in this manner, but wildcards such as x.*.y.z will fall back to the `NoDelay` behavior."